episodes/structure-schema.Rmd

```diff
@@ -0,0 +1,464 @@
+"**ax**" describes the **biomechanical acceleration of the test person on the x axis** in **m/s²**.
+"**ay**" describes the **biomechanical acceleration of the test person on the y axis** in **m/s²**.
+"**az**" describes the **biomechanical acceleration of the test person on the z axis** in **m/s²**.
+"**scr**" is a **boolean** indicating a **detected scream of the test person**.
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: challenge
+## Challenge 3: Write JSON metadata record
+
+You have manually marked up the relevant information in the README. However, your project requires you to provide metadata in the form of a **machine-readable JSON metadata record**. The project provides you with a simple example JSON object (remember, that curly braces hold objects):
+
+```json
+{
+  "key":"value",
+  "key":"value"
+}
+``` 
+
+- Based on the information identified in the README; write a well-formed, descriptive JSON object.
+- Collaboratively, find suitable keys (aka variable names) to your values.
+- You may want to use some <a href="https://jsonformatter.org/" target="_blank">JSON formatter web service</a> to check and pretty-print your output.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: hint
+
+**Keep in mind**, that values in JSON must be one of the following data types:
+
+- a string `""`
+- a number `42`
+- a boolean `True`
+- null `null`
+- an array `[]`
+- an object `{}`
+
+**Example:**
+```json
+{
+    "key":"value",
+    "aString":"string",
+    "anInteger":5,
+    "aFloat":0.5,
+    "aBoolean":true,
+    "anArray": ["item1", "item2", "item3"],
+    "anObject": {
+        "key1":"value1",
+        "key2":"value2",
+        "key3":"value3"
+    }
+}
+```
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: solution
+This is just one (of many) valid solutions.
+
+```json
+{
+  "fileName": "trainingObject.csv",
+  "abstract": "The data describes the biomechanical acceleration and screams detected of a test person during the ride of the roller coaster \"Flight of the Bat\" in Gotham City.",
+  "format": "text/csv",
+  "date": "2022-02-28",
+  "creator": [
+    {
+      "creatorName": "Bruce Wayne",
+      "creatorAffiliation": "Institute for Vigilance and Nightly Motion - Justice League"
+    },
+    {
+      "creatorName": "Selina Kyle",
+      "creatorAffiliation": "Institute for Vigilance and Nightly Motion - Justice League"
+    }
+  ],
+  "experimentalParameters": {
+    "testRide": {
+      "rideName": "Flight of the Bat",
+      "location": "Gotham City, New Jersey",
+      "rideType": "roller coaster"
+    },
+    "testPerson": {
+      "sex": "male",
+      "height": 180
```

Comment:
Even though there's multiple correct solutions, I think that the weight of the test person should be included in the JSON metadata record, since it was marked as valuable information in the previous example.

Also, maybe include units for the height, specially since it was converted from imperial to metric.